cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/CassandraClusterInfo.java

```diff
@@ -535,6 +548,15 @@ protected boolean isReplacement(RingEntry ringEntry)
         return false;
     }
 
+    protected CassandraBridge bridge()
+    {
+        if (bridge == null)
+        {
+            bridge = CassandraBridgeFactory.get(getLowestCassandraVersion());
```

Comment:
underlying ConcurrentHashMap in factory guarantees single bridge initialization here, so no need for synchronization

Reply:
yeah, having a local cache saves look ups from the map. But lookups are fast anyway, so I think we can remove the local field used as a cache